### ISSUE TITLE
Meaning of SSE and SSR in models.ols()

### ISSUE THREAD

**Issue by mcclella** [jstat/jstat#286]
The values returned for SSE and SSR are the reverse of the usual uses of those abbreviations.  In most textbooks, 
    SST = SSR + SSE
where SST is the Toal sum of squares (i.e., Sum(X - Bar)^2), SSE is the remaining Sum of Squared ERROR, and SSR is the amount that the original SST was REDUCED.  models.ols() reports SSR evidently as the SSResiduals which isn't wrong, just unusual.  and it leaves SSE without a clear meaning.  IMHO, it would be better to reverse those two.

**Comment by bklingen** [comment]
Yes, agreed, the two should be swapped. SSE is typically the sum of squares **Error**, from which the MSE (mean squared error) is computed.

SSR is typically the sum of squares Regression.

Currently, however, jstat returns SSR as the sum of squares Errors and SSE as the sum of squares Regression.

Flipping the notation would make sense.
